README.md

```diff
@@ -1301,6 +1310,203 @@ By default, it will revert all the clusters present in the backup folder
 
 [[back to top](#databricks-labs-ucx)]
 
+# Common Challenges and the Solutions
+Users might encounter some challenges while installing and executing UCX. Please find the listing of some common challenges and the solutions below.
```

Comment:
please add `[[back to top](#databricks-labs-ucx)]` in the end of every section, so that it's consistent with the rest of the document

Reply:
Done